build-system/global-configs/canary-config.json

```diff
@@ -14,7 +14,6 @@
 
   "canary": 1,
 
-  "a4aFastFetchAdSenseLaunched": 0,
   "a4aFastFetchDoubleclickLaunched": 0,
```

Comment:
how about this?

Reply:
sorry, missed that one!  fixed in e2b792a14